features/networking/multus.feature

```diff
@@ -1008,3 +1008,114 @@ Feature: Multus-CNI related scenarios
     Then the output should contain:
       | 192.168.22.2     |
 
+  # @author anusaxen@redhat.com
+  # @case_id OCP-22504
+  @admin
+  Scenario: The multus adminssion controller should be able to detect that the pod is using net-attach-def in other namespaces when the isolation is enabled
+    Given I create 2 new projects
+    # Create the net-attach-def via cluster admin
+    When I run oc create as admin over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/macvlan-bridge.yaml" replacing paths:
+      | ["metadata"]["name"]      | macvlan-bridge-25657    |
+      | ["metadata"]["namespace"] | <%= project(-1).name %> |    
+    Then the step should succeed
+    Given I use the "<%= project(-2).name %>" project
+    # Create a pod in new project consuming net-attach-def from 1st project
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/Pods/1interface-macvlan-bridge.yaml" replacing paths:
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | macvlan-bridge-25657 |
+    Then the step should succeed
+    And evaluation of `@result[:response].match(/pod\/(.*) created/)[1]` is stored in the :pod_name clipboard
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :describe client command with:
+      | resource | pods               |
+      | name     | <%= cb.pod_name %> |
+    Then the step should succeed
+    And the output should contain:
+      | cannot find get a network-attachment-definition |
+    """
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24492
+  @admin
+  Scenario: Create pod with Multus ipvlan CNI plugin	
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    And I store all worker nodes to the :nodes clipboard
+    And the default interface on nodes is stored in the :default_interface clipboard
+    #Storing default interface mac address for comparison later with pods macs
+    Given I use the "<%= cb.nodes[0].name %>" node
+    And I run commands on the host:
+      | ifconfig <%= cb.default_interface %> |
```

Comment:
we really should stop using ifconfig, it's been deprecate since like 2009.

https://bugzilla.redhat.com/show_bug.cgi?id=687920#c1

Reply:
And thanks for that warning. Changed it to ```ip addr```